cg-spec/editors_draft.html

```diff
@@ -1420,6 +1420,112 @@ <h2>isTRIPLE</h2>
 
       </section>
 
+      <section class="informative">
+        <h2>Examples and Discussion</h2>
+
+        <p>By the evaluation semantics of SPARQL-star as defined in <a href="#evaluation-semantics"></a>, the five new built-in functions defined above can be used in exactly the same manner as <a data-cite="SPARQL11-QUERY#SparqlOps">the functions defined in the SPARQL 1.1 spec</a><!--, which includes using them within GRAPH clauses that focus on querying named graphs with a queried dataset-->. For instance, the following SPARQL-star query retrieves every <a>asserted triple</a> that, both, is contained in the <a>default graph</a> of the queried <a>RDF-star dataset</a> and has an <a>embedded triple</a> as its <a>subject</a>.</p>
+
+        <pre data-transform="updateExample"
+             data-content-type="application/x-sparql-star-query"
+             class="nohighlight example"
+        >
+          <!--
+          SELECT ?t WHERE {
+            ?s ?p ?o .
+            BIND( TRIPLE(?s,?p,?o) AS ?t )
+            FILTER( isTRIPLE(SUBJECT(?)) )
+          }
+          -->
+        </pre>
+
+        <p>Instead of accessing the subject of the triples in the FILTER after the BIND clause, as done in the previous query, a semantically equivalent variation of this query may apply the FILTER directly on the variable&nbsp;<code>?s</code> as follows.</p>
```

Comment:
```suggestion
        <p>Instead of accessing the subject of the triples in the FILTER after the BIND clause, as was done in the previous query, a semantically equivalent query may apply the FILTER directly on the variable&nbsp;<code>?s</code>, as follows:</p>
```